server/queue/queue-activemq/src/test/java/org/apache/james/queue/activemq/ActiveMQMailQueueTest.java

```diff
@@ -71,12 +81,61 @@ protected static BrokerService createBroker() throws Exception {
         return broker;
     }
 
+    @AfterEach
+    public void tearDown() throws Exception {
+        broker.stop();
+    }
+
+    @Override
+    public MailQueue getMailQueue() {
+        return mailQueue;
+    }
+
+    @Override
+    public ManageableMailQueue getManageableMailQueue() {
+        return mailQueue;
+    }
+
+    @Test
+    @Override
+    @Disabled("JAMES-2295 Disabled as test was dead-locking")
+    public void dequeueCanBeChainedBeforeAck() {
+
+    }
+
+    @Test
     @Override
-    protected JMSMailQueue createQueue(ConnectionFactory factory, MailQueueItemDecoratorFactory mailQueueItemDecoratorFactory, String queueName) {
-        return new ActiveMQMailQueue(factory, mailQueueItemDecoratorFactory, queueName, useBlobMessages(), new NoopMetricFactory());
+    @Disabled("JAMES-2295 Disabled as test was dead-locking")
+    public void dequeueCouldBeInterleavingWithOutOfOrderAck() {
+
+    }
+
+    @Test
+    @Override
+    @Disabled("JAMES-2301 Per recipients headers are not attached to the message.")
```

Comment:
idem?

Reply:
Still not. Maybe in another PR, if you think it do is important.